Error while checking build/O2/fullCI for 60ec119644c3479889674667cce3b66337f33407 at 2024-04-16 11:48:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'


## sw/BUILD/QualityControl-latest/log
/sw/SOURCES/QualityControl/v1.141.1/v1.141.1/Modules/CTP/src/RawDataReaderCheck.cxx:387:73: error: no matching function for call to 'o2::ctp::CTPRunManager::getConfigFromCCDB(long int&, std::string&)'
ninja: build stopped: cannot make progress due to previous errors.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/12974/60ec119644c3479889674667cce3b66337f33407/build_O2_fullCI/pretty.html).

--- a/DataFormats/Detectors/CTP/include/DataFormatsCTP/RunManager.h
+++ b/DataFormats/Detectors/CTP/include/DataFormatsCTP/RunManager.h
@@ -39,7 +39,8 @@ class CTPRunManager
   void printActiveRuns() const;
   int saveRunScalersToCCDB(int i);
   int saveRunConfigToCCDB(CTPConfiguration* cfg, long timeStart);
-  static CTPConfiguration getConfigFromCCDB(long timestamp, std::string run, bool ok);
+  static CTPConfiguration getConfigFromCCDB(long timestamp, std::string run, bool& ok);
+  static CTPConfiguration getConfigFromCCDB(long timestamp, std::string run);
   CTPRunScalers getScalersFromCCDB(long timestamp, std::string, bool& ok);
   int loadScalerNames();
   // void setCCDBPathConfig(std::string path) { mCCDBPathCTPConfig = path;};

--- a/DataFormats/Detectors/CTP/src/RunManager.cxx
+++ b/DataFormats/Detectors/CTP/src/RunManager.cxx
@@ -306,7 +306,7 @@ int CTPRunManager::saveRunConfigToCCDB(CTPConfiguration* cfg, long timeStart)
   LOG(info) << "CTP config  saved in ccdb:" << mCCDBHost << " run:" << cfg->getRunNumber() << " tmin:" << tmin << " tmax:" << tmax;
   return 0;
 }
-CTPConfiguration CTPRunManager::getConfigFromCCDB(long timestamp, std::string run, bool ok)
+CTPConfiguration CTPRunManager::getConfigFromCCDB(long timestamp, std::string run, bool& ok)
 {
   auto& mgr = o2::ccdb::BasicCCDBManager::instance();
   mgr.setURL(mCCDBHost);
@@ -322,6 +322,16 @@ CTPConfiguration CTPRunManager::getConfigFromCCDB(long timestamp, std::string ru
     ok = 1;
   }
   return *ctpconfigdb;
+}
+CTPConfiguration CTPRunManager::getConfigFromCCDB(long timestamp, std::string run)
+{
+  bool ok;
+  auto ctpconfig = getConfigFromCCDB(timestamp,run,ok);
+  if(ok == 0) {
+    LOG(error) << "CTP config not in CCDB";
+    return CTPConfiguration();
+  }
+  return ctpconfig;
 }
 CTPRunScalers CTPRunManager::getScalersFromCCDB(long timestamp, std::string run, bool& ok)
 {

--- a/DataFormats/Detectors/CTP/src/RunManager.cxx
+++ b/DataFormats/Detectors/CTP/src/RunManager.cxx
@@ -326,8 +326,8 @@ CTPConfiguration CTPRunManager::getConfigFromCCDB(long timestamp, std::string ru
 CTPConfiguration CTPRunManager::getConfigFromCCDB(long timestamp, std::string run)
 {
   bool ok;
-  auto ctpconfig = getConfigFromCCDB(timestamp,run,ok);
-  if(ok == 0) {
+  auto ctpconfig = getConfigFromCCDB(timestamp, run, ok);
+  if (ok == 0) {
     LOG(error) << "CTP config not in CCDB";
     return CTPConfiguration();
   }

